### ISSUE TITLE
Spelling Errors in User guide

### ISSUE THREAD

**Issue by fisher729** [nvaccess/nvda#7371]
Hi.

In section 14.1:
Tels NVDA to start with log level set to info and to write its log to c:\nvda.log

I noticed this using a OneCore voice :)

**Comment by josephsl** [comment]
Hi, any other spelling errors? Looks like another round of doc fixes is in order… Thanks.

**Comment by PratikP1** [comment]
Thanks for the challenge, @josephsl.  Here is my list. I have not gone through the document for grammatical accuracies with a few minor exceptions.

Here are a couple of things to note in NVDA next-14168,fb78b7e9 User Guide.

* If American English is standard, then the spelling of “license” should be substituted for “licence” Throughout the document. If U.K.-based English is used throughout, then feel free to ignore suggestions that relate to English variances below.
* In Section 4.1, term “extra long” is grammatically incorrect. It should be “extra-long” (with a hyphen). “If it is taking an extra long time, NVDA should say ‘Loading NVDA. Please wait...’”
* In section 4.6 “self voicing” should be “self-voicing. “Press this command again to disable self voicing mode.”
* In Section 8.1 (table), row 21, column 2 should read “separator” and not “seperator.”
* Multiple places in the users Guide, “combobox” is used as opposed to “combo box.” There are other places where “combo box” is used. No matter what, this should be consistent.
* Depending on the English variant chosen, the word “realising” could be problematic in section 10.1.6.
* In section 10.1.7, the word “controlled” should be “controlled.” If the "play audio coordinates when mouse moves" checkbox is checked, then checking this checkbox means that the volume of the audio coordinates beeps is controled by how bright the screen is under the mouse.
* In Section 10.1.10, multiple occurrences of the word “Chinese” should be capitalized. “Having this option on for pictographic input methods such as chinese New ChangJie or Boshiami is useful …” “However, for phonetic input methods such as chinese New Phonetic, it may be more useful to turn this option off.”
* In section 10.1.10, the word “Japanese” should also be capitalized. “This option may be useful for Korean and japanese input methods.”
* In section 10.1.12, spell the word “organised” based on English chosen—American or not. “The document formatting options are organised into groups.”
* In Section 10.1.12, add a full stop/period after “etc”. “Paragraph indentation (e.g. hanging indent, first line indent) Line spacing (single, double etc)”
* In section 10.4, “appModules” should be “app Modules.” There are multiple occurrences of this. If “appModules” is considered the standard, then correct the occurrences of “app Modules” in other parts of the guide. “Portable versions of NVDA store all settings, custom appModules and custom driver …”
* In section 11.1, the word “occurred” should be “occurred.” “… logging output that has occured up until now from when you last started NVDA.”
* In section 14.1, The word “etc” should have a full stop/period after it. “No sounds, no interface, no start message etc”

**Comment by josephsl** [comment]
CC @JCSTeh, @feerrenrut and other documentation specialists.

**Comment by jcsteh** [comment]
Thanks @PratikP1 for digging through this. :) A few clarifications:

[**@PratikP1**](https://github.com/PratikP1) commented on [11 Jul 2017, 04:22 GMT+10](https://github.com/nvaccess/nvda/issues/7371#issuecomment-314192062 "2017-07-10T18:22:38Z - Replied by Github Reply Comments"):
> * If American English is standard, then the spelling of “license” should be substituted for “licence” Throughout the document.

I only see one occurrence of "licence" (with a c). We've standardised on U.S. English for the NVDA project, so this should be changed to "license" (with an s).

As a side note, NV Access, as an Australian organisation, standardises on British English for our website, communication, promotional material, books, etc. However, programming languages tend to standardise on U.S. English, so we use U.S. English for the code. Originally, some of the documentation was British English, but it made sense to be consistent between the code and documentation, so we transitioned (albeit haphazardly) to U.S. English. This might help to explain the confusion surrounding this from time to time.

> * Multiple places in the users Guide, “combobox” is used as opposed to “combo box.”

All of these should be changed to "combo box" (with a space).

> * Depending on the English variant chosen, the word “realising” could be problematic in section 10.1.6.

It should be "realizing" (with a z).

> * In section 10.1.12, spell the word “organised” based on English chosen

It should be "organized".

> * In Section 10.1.12, add a full stop/period after “etc”. “Paragraph indentation (e.g. hanging indent, first line indent) Line spacing (single, double etc)”

There should also be a comma before "etc.".

> * In section 10.4, “appModules” should be “app Modules.”

Actually, I think we should change this to talk about "settings and add-ons". This is more familiar to users, and the term "add-ons" covers app modules, drivers and also global plugins.

> * In section 14.1, The word “etc” should have a full stop/period after it. “No sounds, no interface, no start message etc”

Also comma before "etc.".

**Comment by josephsl** [comment]
Hi, will work on it tonight, along with modifying what’s new (once again) and adding “Welcome to NVDA!” at the beginning of the user guide to make it even friendlier (been meaning to propose this). Thanks.

**Comment by jcsteh** [comment]
I'd suggest splitting the "welcome" addition into a separate PR because
it's unrelated to corrections and thus subject to separate criteria. Thanks.

**Comment by fisher729** [comment]
Hi.

The word 'balloon' is incorrectly spelled 'baloon' in Section 8.1 in the Master user guide.

**Comment by ehollig** [comment]
Confirmed. CC @jcsteh
